Assets/_BForBoss/_Analytics/Scripts/PerigonAnalytics.cs

```diff
@@ -43,19 +44,25 @@ public class PerigonAnalytics : IPerigonAnalytics
 
         public static PerigonAnalytics Instance => _instance;
 
-        #region CONSTRUCTORS
+        #region CONSTRUCTORS / SETTERS
         static PerigonAnalytics()
         {
         }
 
         private PerigonAnalytics()
         {
         }
+
+        public void SetUsername(String username)
+        {
+            Mixpanel.People.Set("$name", username);
```

Comment:
Is the $ so that it overrides mixpanel default name

Reply:
the $ is just part of the field name for mixpanel so that I can set its value